pkg/cmd/server/origin/run_components.go

```diff
@@ -569,7 +571,17 @@ func (c *MasterConfig) RunUnidlingController() {
 }
 
 func (c *MasterConfig) RunTemplateController() {
```

Comment:
Follow the pattern like https://github.com/openshift/origin/blob/master/pkg/cmd/server/origin/controller/apps.go#L28